databricks/koalas/namespace.py

```diff
@@ -406,11 +406,12 @@ def read_csv(
                 kdf[col] = kdf[col].astype(dtype)
 
     if squeeze and len(kdf.columns) == 1:
-        return first_series(kdf)
-    return kdf
+        return cast(DataFrame, first_series(kdf))
```

Comment:
This shouldn't be casted. `read_csv` CAN return `Series` when `squeeze is True`.

Reply:
Fixed in 72e5819. Thanks!